### PR TITLE
[configuration]exposeBunlesMetricsInPrometheus

### PR DESCRIPTION
**Pull request by gaozhangmin**
Issue #12366  support expose bundles metrics to prometheus, but forget add configuration `exposeBunlesMetricsInPrometheus` to broker.conf.

### Documentation

Check the box below and label this PR (if you have committer privilege).

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [x] `no-need-doc` 
  
doc already added in #12825  
- [ ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

**Comment by gaozhangmin** [comment]
@Anonymitaet  PTAL

File: conf/broker.conf [hunk 1]
```diff
@@ -1232,6 +1232,9 @@ exposeManagedCursorMetricsInPrometheus=false
 # Set it to 0 to disable timeout.
 metricsServletTimeoutMs=30000
 
+#Enable or disable the broker bundles metrics. default is false
```
Comment (Anonymitaet) [inline conf/broker.conf:1235]:
```suggestion
# Enable or disable broker bundles metrics. The default value is false.
```

File: conf/broker.conf [hunk 2]
```diff
@@ -1232,6 +1232,9 @@ exposeManagedCursorMetricsInPrometheus=false
...
+exposeBunlesMetricsInPrometheus
```
Comment (Anonymitaet) [inline conf/broker.conf:1236]:
```suggestion
exposeBunlesMetricsInPrometheus=false
```
this shows the default value, right?

**Comment by Anonymitaet** [comment]
@gaozhangmin since this change to `broker.conf` can be regarded as a kind of instructions (doc), I label this PR w/ `doc.

**Review by Anonymitaet — APPROVED** [review]

**Review by Jason918 — APPROVED** [review]

File: conf/broker.conf [hunk 3]
```diff
@@ -1232,6 +1232,9 @@ exposeManagedCursorMetricsInPrometheus=false
 # Set it to 0 to disable timeout.
 metricsServletTimeoutMs=30000
 
+# Enable or disable broker bundles metrics. The default value is false.
+exposeBunlesMetricsInPrometheus=false
```
Comment (codelipenghui) [inline conf/broker.conf:1236]:
```suggestion
exposeBundlesMetricsInPrometheus=false
```

Comment (codelipenghui) [reply conf/broker.conf:1236]: We don't have a release that contains https://github.com/apache/pulsar/pull/12366, so you can fix the typo directly.

**Comment by gaozhangmin** [comment]
fix typo PTAL again. @codelipenghui

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**